test/library/encryption/helpers.go

```diff
@@ -227,7 +233,7 @@ func ForceKeyRotation(t testing.TB, updateUnsupportedConfig UpdateUnsupportedCon
 		return err
 	}
 
-	return retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+	return onErrorWithTimeout(wait.ForeverTestTimeout, retry.DefaultBackoff, orError(errors.IsConflict, transientAPIError), func() error {
```

Comment:
do we need `onError` since `IsConflict` is already considered `transientError` ?

Reply:
IsConflict is not transient